dev/tests/api-functional/phpunit_soap.xml.dist

```diff
@@ -19,6 +19,10 @@
         <testsuite name="Magento SOAP web API functional tests">
             <directory suffix="Test.php">testsuite</directory>
             <!-- <exclude>testsuite/Magento/GraphQl</exclude> -->
+            <directory suffix="Test.php">../../../app/code/*/*/Test/Api</directory>
+        </testsuite>
+        <testsuite name="msi">
```

Comment:
Looks like we need to remove this testsuite
It is related only to MSI repository

Reply:
Done